src/Guild.js

```diff
@@ -25,6 +25,35 @@ class Guild {
     autoBind(this);
   }
 
+  guildRoleCreate (roleId) {
+    this.roles[this.roles.name] = roleId;
+  }
+
+  guildGetKeyByValue (value) {
```

Comment:
`GetKeyByValue` could be talking about any object. A function name should accurately describe what is happening so future contributors can go off the intellisense without needing to check what is happening in the function. I'd suggest `getRoleNameByID`.

Reply:
I didn't think of that. Easy to get a bit stuck in the logic alone rather than thinking that other people is also going to have to interpret this at a later point as well 😅 Fixed this now, i definitely will keep this in mind at a later point.